src/_guides/language/effective-dart/design.md

```diff
@@ -1555,14 +1555,19 @@ bool convertToBool(dynamic arg) {
 ### DO use `Future<void>` as the return type of asynchronous members that do not produce values.
 
 When you have a synchronous function that doesn't return a value, you use `void`
-as the return type. The asynchronous equivalent is `Future<void>`.
+as the return type. The asynchronous equivalent for a method that doesn't
+produce a value, but that the caller might need to await, is `Future<void>`.
```

Comment:
Maybe "await on"? @kwalrath WDYT?

Reply:
I had to look this up to be sure, but "await" as a verb already has an implied preposition—it's the equivalent of "wait for". So we wouldn't need the "on".